messaging-activemq/injection/src/test/java/org/wildfly/extension/messaging/activemq/deployment/injection/AbstractJMSContextTestCase.java

```diff
@@ -0,0 +1,80 @@
+package org.wildfly.extension.messaging.activemq.deployment.injection;
```

Comment:
All new classes should have a copyright header added with the year set to the current year,